lstchain/reco/r0_to_dl1.py

```diff
@@ -624,22 +624,29 @@ def r0_to_dl1(
                                           use_main_island=True)
                     image = event.dl1.tel[telescope_id].image
 
-                    if ('lh_fit_config' in config.keys()
-                       and dl1_filled['n_pixels'] is not 0
-                       and dl1_filled['n_pixels'] < 1000):
-                        is_saturated = np.any(image > config['lh_fit_config']['n_peaks'])
-
-                        if not is_saturated: #reject computationnally expensive events which would be poorly estimate with the selected value of n_peak TODO : improve to not reject events
-
-                            dl1_filled = get_dl1_lh_fit(event,
-                                                        subarray,
-                                                        telescope_id,
-                                                        image=image,
-                                                        is_simu=is_simu,
-                                                        normalized_pulse_template=pulse_template,
-                                                        dl1_container=dl1_filled,
-                                                        custom_config=config,
-                                                        use_main_island=True)
+                    if ('lh_fit_config' in config.keys():
+                        if dl1_filled['n_pixels'] is not 0
+                           and dl1_filled['n_pixels'] < 1000):
+                            is_saturated = np.any(image > config['lh_fit_config']['n_peaks'])
+
+                            if not is_saturated: #reject computationnally expensive events which would be poorly estimate with the selected value of n_peak TODO : improve to not reject events
+                                dl1_filled.lhfit_call_status = "Not processed : Error in function"
```

Comment:
If you envision a possibility of failure, the next code block with a call to `get_dl1_lh_fit` function has to be wrapped into try-except block

Reply:
Not sure about the except argument that I used, but it seems quite generic. From ctapipe:
'''
class HillasParameterizationError(RuntimeError):
    pass
'''

Reply:
see my other comment